doc/syntax.bnf

```diff
@@ -133,7 +133,7 @@ reserved "why3"
   | "focus" <nat_lit>
   | "print"
   | "proofterm"
-  | "why3" <string_lit>?
+  | "why3" <string_lit>?
```

Comment:
Why do you change anything here? On the other hand, you should add something about "inductive". Idem in doc/sections/commands.md (just saying that it generates new symbols; details will be given later).

Reply:
I think it's just a bug from github. As you can see the two lines are the same.
OK I won't forget that.

Reply:
They look the same but perhaps they are not (a different space character).

Reply:
Oh I know!
Yes, before this space character wasn't it. Probably someone used Ctrl + space.
Now it's OK.